Vagrantfile

```diff
@@ -18,12 +18,22 @@ vagrant_boxes = {
   "focal-m1" => "https://app.vagrantup.com/bento/boxes/ubuntu-20.04/versions/202404.23.0/providers/parallels/arm64/vagrant.box",
   "jammy" => "http://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64-vagrant.box",
   "jammy-libvirt" => "https://vagrantcloud.com/generic/boxes/ubuntu2204/versions/4.3.12/providers/libvirt/amd64/vagrant.box",
-  "jammy-m1" => "https://app.vagrantup.com/bento/boxes/ubuntu-22.04/versions/202407.22.0/providers/parallels/arm64/vagrant.box",
   "dummy" => "https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box",
 }
+# boxes are often provider agnostic, and ideally the latest version would be
+# fine, but it's also useful to keep a record of specific versions that have
+# actually been seen to work
+vagrant_box_aliases = {
+  "jammy-arm" => ["bento/ubuntu-22.04", ">=0"],
+  "jammy-arm-vmware" => ["bento/ubuntu-22.04", "202401.31.0"],
+  "jammy-arm-parallels" => ["bento/ubuntu-22.04", "202407.22.0"],
+}
 vagrant_box = (ENV['VAGRANT_BOX'] || DEFAULT_BOX)
 username = (ENV['VAGRANT_USERNAME'] || "vagrant")
 
+# maybe there is a way to query vagrant for the selected box architecture?
```

Comment:
Vagrant itself seems smart enough to [detect the host architecture](https://developer.hashicorp.com/vagrant/docs/vagrantfile/machine_settings#config-vm-box_architecture) but IDK how to query for what it selected